test/src/fictionalpancake/turbospork/ColorTest.java

```diff
@@ -0,0 +1,49 @@
+package fictionalpancake.turbospork;
+
+import fictionalpancake.turbospork.paint.Color;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class ColorTest {
+    @Test
+    public void testRGB() {
+        Random rand = new Random();
+        int r = rand.nextInt(256);
+        int g = rand.nextInt(256);
+        int b = rand.nextInt(256);
+        Color c = new Color(r, g, b);
+        Assert.assertEquals(c.getRed(), r);
+        Assert.assertEquals(c.getGreen(), g);
+        Assert.assertEquals(c.getBlue(), b);
+    }
+
+    @Test
+    public void testRGBA() {
+        Random rand = new Random();
+        int r = rand.nextInt(256);
+        int g = rand.nextInt(256);
+        int b = rand.nextInt(256);
+        int a = rand.nextInt(256);
+        Color c = new Color(r, g, b, a);
+        Assert.assertEquals(c.getRed(), r);
+        Assert.assertEquals(c.getGreen(), g);
+        Assert.assertEquals(c.getBlue(), b);
+        Assert.assertEquals(c.getAlpha(), a);
+    }
+
+    @Test
+    public void testNumber() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork/file/2610964921/issues/source?bid=3248828&fileBranchId=3310821#l37)